src/components/common/Buttons/BasicButton.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
@abayo-luc, I'd like to know why you're not using the shared components that we've created.
If you already have created the others before, you can delete one of them and communicate which ones the team shall keep using just to keep consistency.

Reply:
We have a different kind of buttons, with different styles and properties. The one I created is FormButtons, which somehow differ from the one in Button.js